cleanslate/doma/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
+    url(r'^chore/(?P<pk>[-\w]+)/edit/$', views.edit_chore_deadline, name = 'edit-chore-deadline'),
```

Comment:
I'm going to do more reading on this but I'm wondering if the ID comes after edit as I've found an example like this:
`r'^book/edit/(?P<object_id>\d+)/$', create_update.update_object, book_info)`
they also made a big deal about the \d+ at the end